lib/dependencies.js

```diff
@@ -0,0 +1,18 @@
+import GetPlan from './use-cases/getPlan';
+import CreatePlan from './use-cases/createPlan';
+import PlanGateway from './gateways/planGateway';
+import AWS from 'aws-sdk';
+
+const config = {
+  region: 'localhost',
+  endpoint: 'http://localhost:8000',
+  accessKeyId: 'foo',
+  secretAccessKey: 'bar'
```

Comment:
Do we want these as environment variables, or conditional based on the environment maybe?

Reply:
ah yep i missed this, will take a look